File: backend/core-service/src/main/java/com/pgc/stress_predict/application/dto/request/UsuarioFormRequest.java
package com.pgc.stress_predict.application.dto.request;

import java.time.LocalDate;

public record UsuarioFormRequest(
        String nombre,
        String apellido,
        Integer dni,
        LocalDate fechaNacimiento,
        String telefono,
        String email,
        String contrasenia,
        Float horasEstudioDia,
        Float horasExtracurricularDia,
        Float horasSuenoDia,
        Float horasSocialDia,
        Float horasActividadFisicaDia,
        Float promedioCalificaciones
) {
}

File: Frontend/app/src/main/java/com/pgc/stresspredict/data/model/request/UsuarioFormRequest.kt
package com.pgc.stresspredict.data.model.request

data class UsuarioFormRequest(
    val nombre: String,
    val apellido: String,
    val dni: Int?,
    val fechaNacimiento: String?, // Usamos String en lugar de LocalDate
    val telefono: String?,
    val email: String,
    val contrasenia: String,
    val horasEstudioDia: Float?,
    val horasExtracurricularDia: Float?,
    val horasSuenoDia: Float?,
    val horasSocialDia: Float?,
    val horasActividadFisicaDia: Float?,
    val promedioCalificaciones: Float?
)